nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/cluster/ZooKeeperClientConfig.java

```diff
@@ -133,6 +207,30 @@ private static int getTimePeriod(final NiFiProperties nifiProperties, final Stri
         }
     }
 
+    private static boolean getClientSecure(final NiFiProperties nifiProperties, final String propertyName, final boolean defaultValue) {
```

Comment:
It seems like it might be better to encapsulate this property evaluation logic using a new method on NiFiProperties

Reply:
Done.